src/render.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-cycle */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './App/App';
+
+import { textAreaFunc, updateTextaAreaFunc } from './redux/state';
+
+const renderFunc = (state) => {
+  ReactDOM.render(<App state={state} textAreaFunc={textAreaFunc} updateTextaAreaFunc={updateTextaAreaFunc} />, document.getElementById('root'));
```

Comment:
Consider not making such long lines of code :)
This way it looks better and more readable.
![image](https://user-images.githubusercontent.com/25229823/77894410-e9f75380-727d-11ea-96e3-82a37807579c.png)

Reply:
Sure it's realy more readable